test/extended/util/client.go

```diff
@@ -119,18 +132,47 @@ func NewCLI(project, adminConfigPath string) *CLI {
 
 // NewCLIWithoutNamespace initialize the upstream E2E framework without adding a
 // namespace. You may call SetupProject() to create one.
-func NewCLIWithoutNamespace(project string) *CLI {
+func NewCLIWithoutNamespace(wsPrefix string) *CLI {
```

Comment:
@Phaow i think we could change the comments too to reflect what is in our code ?

Reply:
Sure. fixed.